### ISSUE TITLE
Races: MotM style updates (Languages, ASIs, and More!)

### ISSUE THREAD

**Issue by Tyloris** [WC5E/Warcraft-5e-Conversion#825]
MotM has made a number of things "standard."

1. Every race gets either +2/+1 or +1/+1/+1 to any ability scores of their choice.
2. Every race gets Common and "one other language that you and your DM agree is appropriate for your character." 
3. A race's creature type is listed in their racial traits, as well as any subtypes.
4. Lifespans are no longer listed in the race description, instead assuming that you live for about a century unless otherwise stated. Note that it makes no remark on how long it takes to reach maturity, and when the race is listed, the age portion is put into the general race description instead it being a racial trait.
5. Alignment is no longer listed.
6. Height and Weight is no longer listed, instead stating that all player characters fall in the same height and weight range as humans in our world--yes, even centaurs and gnomes.

Now, points 1 and 2 are likely just to streamline character creation. I, personally, like the specific ASIs and language traits and would rather keep them. I think they are important guidelines for character creation. However, I wouldn't mind there being a variant rule option at the beginning of the races chapter stating that they are intended as guidelines and that players can instead use the MotM format if they so choose.

Point 3 is easy enough to implement and probably should have been done so in the beginning.

Points 4, 5, and 6... bother me. 4 and 6 more so than 5. Part of their reasoning seems to be that they were trying to reduce each of the races to fit onto a single page--no more, no less--so much so that a great deal of background information is left out of the race. Compare the specific entries on aasimar in [Volo's Guide](http://online.anyflip.com/awzb/monr/mobile/index.html#p=105) to them in [MotM](https://anyflip.com/fqhgb/qrus/).

Another part of their reasoning is that MotM is that A) these are supplement races, instead of core races, and thus have less lore (like how vulpera don't have near the amount of lore of the orcs), and B) the book is likely designed to be system agnostic, able to be used from Forgotten Realms to Dragonlance to Eberron. So the lore on each is pretty barebones compared to something as setting specific as our project.

If we do want to remove the Age, Alignment, and/or Size, I'd suggest moving some or all of the details to our Chapter 3, which is currently just called 'New Backgrounds.' Things like Age and Size would be pretty appropriate there, as they probably rarely come up in non-background situations and can pretty easily be formatted into a couple tables. Plus Size hasn't been properly integrated in the github document, with some races like draenei having random height and weight entries, while others like orcs do not. Also, #768

**Comment by OmNomDom308** [comment]
I agree with most of this. Im fine/like the racials and having the specific languages personally, and think its fine here bc we dont have to be as setting agnostic like WotC aims to. 

3 I agree should be implemented

I like having Age, adds alot of flavor and doesnt add THAT much space in the grand scheme of things. Alignment Im a bit in the air, just bc alot of that reasonings WoTC did that also came with a general swathe of removing set alignments on most things for their reasons. 

Size I think they removed mostly bc They now give PCs the option to be Small or Medium within one race, so a height chart kinda buggers that. A table for it somewhere else could work to keep it more simple though
